### DISCUSSION TITLE
Why net6.0 and not netstandard2.0?

### DISCUSSION THREAD

**Discussion by m1stegmann** [nats-io/nats.net#52] [General]
Why not netstandard2.0. This seems very unusual to me. Besides, one would then also support net7.0, which would be very desirable.
Even Span<> and ReadOnlySpan<> could be used via the Nuget package System.Memory in netstandard2.0. The libary would be usable even under Net4.x

**Comment by mtmk** [comment]
We plan to follow the LTS support policy as outlined in [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core). For our users who are still using .NET Framework or .NET Core we recommend our [NATS.NET v1 client](https://github.com/nats-io/nats.net).
